Dangerfile

```diff
@@ -296,6 +297,22 @@ def validateSwiftFiles(file)
   end
 end
 
+def checkHardcodedNib(file)
+  File.foreach(file) do |line|
+    line = line.gsub('\n','').strip
+    ext = File.extname(file)
+    case ext
+    when ".xib"
+      if line =~ /<color(?!.*(?:white="1"|white="0\.0"|red="0\.93725490196078431" green="0\.93725490196078431" blue="0\.95686274509803926" alpha="1"|red="0\.93725490199999995" green="0\.93725490199999995" blue="0\.95686274510000002" alpha="1"|red="0\.0" green="0\.0" blue="0\.0"|red="1" green="1" blue="1" alpha="1"|key="textColor" cocoaTouchSystemColor="darkTextColor"\/>|key="titleShadowColor"|cocoaTouchSystemColor|<nil|<view|userDefinedRuntimeAttribute|<string key="text"|image|<\/string>|<label|<document)).*\/>/
```

Comment:
@ghashi just out of curiosity, how did you come up with these values like `0. 95686274509803926`? Are they part of XCode crazy behavior to somehow add them when hardcoded values are present or were these vales specific for some project?